conanfile.py

```diff
@@ -38,6 +38,13 @@ def config_options(self):
     def requirements(self):
         self.requires("libtirpc/1.1.4@bincrafters/stable")
 
+    def system_requirements(self):
+        if tools.os_info.with_yum:
+            packages = ["gettext-devel"]
```

Comment:
I've started a recipe for [gettext](https://github.com/bincrafters/conan-gettext_installer) some time ago. Seems like be time to re-visit that recipe.

Reply:
I cannot reproduce the error that made me add this requirement.
So I've removed the extra requirement.

Reply:
I think conan-gettext `should` be a hybrid project: library + installer.